pkg/registry/core/persistentvolume/storage/storage_test.go

```diff
@@ -19,6 +19,7 @@ package storage
 import (
```

Comment:
Should we explicit enable the feature gate in these tests?

Reply:
I've added it for the modified test (`TestUpdateStatus`). Explicit is better, but is there any other advantage if it's enabled by default now?

Reply:
@jsafrane PTAL again, lgtm was removed. [Here](https://github.com/kubernetes/kubernetes/compare/0fcfea649cda8915b1ceaa886eeea23d903afb55..e42e659955b8aa0165bdf4406a07001d99b38b8a)'s the diff, the followup push is just a rebase.